paymentmethod.go

```diff
@@ -329,6 +330,7 @@ type PaymentMethodOXXO struct {
 
 // PaymentMethodP24 represents the P24 properties.
 type PaymentMethodP24 struct {
+	Bank string `json:"bank"`
```

Comment:
Technically `bank` is an enum so it should have a list of constants but we haven't done that for other `bank` properties on `PaymentMethod` yet so I want to stay consistent (easier for code examples and such) and we can fix with codegen later (or manually if needed).